core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -434,6 +437,44 @@ export class NavigationComponent {
   onClickGetAllVersions() {
     this.workflowVersionService.clickDisplayWorkflowVersions();
   }
+
+  private handleVersionDisplay(): void {
+    this.workflowVersionService
+      .getDisplayParticularVersionStream()
+      .pipe(untilDestroyed(this))
+      .subscribe(displayVersionFlag => {
+        this.particularVersionDate =
+          this.workflowActionService.getWorkflowMetadata().creationTime === undefined
+            ? ""
```

Comment:
hmm maybe try `??` operator?

Reply:
it makes the code longer because need to cast it either way so better do it this way